pkgs/data/fonts/vollkorn/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, stdenv, fetchzip }:
+let
+  pname = "vollkorn";
+  version = "4.105";
+in
+fetchzip {
+  name = "${pname}-${version}";
+  url = "http://vollkorn-typeface.com/download/vollkorn-4-105.zip";
```

Comment:
That url should contain the version variable.

Reply:
oh yeah, that happens when you completely redo a PR /0\.
Does it make sense to make the follow-up on this a `--fixup` commit? Or is a proper commit message preferred, as the fixup won't ever be squashed anyways`

Reply:
#117679